### DISCUSSION TITLE
vscode customization settings presedence

### DISCUSSION THREAD

**Discussion by lmilbaum** [devcontainers/community#50] [Q&A]
There are vscode customizations settings in .devcontainer/devcontainer.json and .vscode/settings.json files. 
Both files are tracked by git. 
Which of the settings takes precedence when spinning devcontainer or/and codespace?

**Comment by bamurtaugh** [comment]
I believe the dev container config would take precedence. Is that what you're observing too? Are you experiencing any specific issues?

**Reply by lmilbaum** [reply]
Thanks for your feedback. Eventually I have managed to resolve the issue I faced. Can't recall what exactly was it.
